GOVERNANCE.md

```diff
@@ -0,0 +1,299 @@
+
+To become a maintainer, [Contributors](#contributors) must express interest to the existing maintainers. The full process is documented in the [MAINTAINERS-GUIDELINES.md](https://github.com/falcosecurity/evolution/blob/master/MAINTAINERS-GUIDELINES.md) file.
+
+The list of current [maintainers](./maintainers.yaml) is automatically updated by our infra (see the [falcosecurity/test-infra](https://github.com/falcosecurity/test-infra) repository).
+
+### Core Maintainers 
+
+Core Maintainers are [Maintainers](#maintainers) of at least one of the [core repositories](#core-repositories).
+The Core Maintainers form a team that drives the direction, values, and governance of the overall project. They also serve as an escalation point for the overall project, and anything not easily managed by the maintainers of each repository.
+
+Defined by:
+ - The OWNERS file `approvers` entry of any [core repository](#core-repositories) (only the OWNERS file in the root directory must be considered).
+
+Requirements:
+ - Same as for [Maintainers](#maintainers).
+
+Responsibilities and privileges:
+ - Overseeing the overall project health and growth.
+ - Speaking on behalf of the project.
+ - Maintaining the brand, mission, vision, values, and scope of the project.
+ - Defining general guidelines for the project.
+ - Administering the [falcosecurity](https://github.com/falcosecurity) GitHub organization.
+ - Administering any assets or services owned or assigned to project.
+ - Handle lincense and copyright issues.
+ - Look out for issues or conflict in any area of the project.
+ - Serve as the last escalation point for an issue that can't be solved by other community roles.
+ - Ability to create committees and delegate powers to them.
+
+The full process to become a Core Maintainer is documented in the [MAINTAINERS-GUIDELINES.md](https://github.com/falcosecurity/evolution/blob/master/MAINTAINERS-GUIDELINES.md) file. 
+
+### Emeritus Maintainers
+
+Emeritus Maintainers are former [Maintainers](#maintainers) of a specific project area (they can still be an active maintainer of another project area). The only path to this role is to be previously listed in the `approvers` entry of an OWNERS file of that project area.
+
+Maintainers who are domain experts over certain areas of the codebase but can no longer dedicate the time needed to handle the responsibilities of reviewing and approving changes are encouraged to add themselves in the OWNERS file `emeritus_approvers` entry.
+
+When an emeritus maintainer returns to being more active, they may be promoted back at the discretion of the current [Maintainers](#maintainers) of the relevant area of the project.
+
+Defined by:
+- The OWNERS file `emeritus_approvers` entry.
+
+Responsibilities:
+ - Lead by example.
+ - May be consulted by [Maintainers](#maintainers).
+ - May serve as sponsors, and commit to the long-term success of The Falco Project.
+
+## Technical Advisory Groups
```

Comment:
It's not clear from this document how TAGs are related to repositories and development. On a naive read, it feels like TAGs have zero ownership/authority at all, since all voting is done by maintainers/core maintainers and TAGs have no say in who those people are.  Please clarify -- not for my sake, but for your own communty's.

Reply:
In our simplistic view, we see TAGs as entities that just cast recommendations and may be consulted by maintainers. 

Since our community has no experience running TAGs (or similar groups), we did not want to introduce any complex structure with high decision powers at this stage. In our opinion, introducing groups or committees with elevated powers would be too risky.

Does that seem reasonable? We are open to any feedback.

Reply:
Do you have an idea of a new TAG that you plan to create after you merge this?

Reply:
We haven't planned the creation of any TAG yet. However, I heard community folks interested in the codebase's specific topics or complex parts. They might want to create groups. 

Two examples that come to mind are: 
- plugins developers who would share best practices and be able to make recommendations on the evolution of the plugin API
- security experts, not necessarily code contributors, that are interested in the security posture of the project and willing to help

In our view, having TAGs allow expert folks from the community to have a voice on specific topics. We also think maintainers will be inclined to seek advice from such experts whenever there are complex decisions.